workflows/argo/prognostic_run_diags.yaml

```diff
@@ -184,7 +184,7 @@ spec:
       command: ["/bin/bash", "-x", "-e", "-c"]
       args:
       - |
-        prognostic_run_diags movie {{inputs.parameters.run}} {{inputs.parameters.output}}
+        prognostic_run_diags movie --n_timesteps 960 {{inputs.parameters.run}} {{inputs.parameters.output}}
```

Comment:
this is just to speed up movies, since they are now computed by default. This will give us 10-day movies for 15-minute data, which is plenty I think.